additionalCSS/footerSection.css

```diff
@@ -152,3 +144,74 @@ hr {
   background: #0d151e;
   border-radius: 10px;
 }
+
+.footerBtn {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
- Kindly the headline content is overlapping with the background image so give it some merging to fix the overlapping and have a better UI 👌
### Figma Design 👇
![2023-05-05_09-18_1](https://user-images.githubusercontent.com/52010789/236390844-2fdb94d0-effe-4f06-9614-bf599579dd99.png)
### your Design 👇
![2023-05-05_09-18](https://user-images.githubusercontent.com/52010789/236390848-2b7a94b7-f339-4388-ac86-91a6b35c5e25.png)